resource/ui/ts/models/stats.ts

```diff
@@ -22,5 +22,61 @@ module Models {
             sys_count: number;
             last_update_nanos: number;
         }
+
+        // This function was adapted from
+        // https://stackoverflow.com/questions/10420352/converting-file-size-in-bytes-to-human-readable
+        export function FormatBytes(bytes: number): string {
+            var thresh = 1024;
```

Comment:
TS doesn't seem to have constants. I'll pull it out of the function at least.  And yes, I'll change the name to kibi. :)